tests/models/test_renderedtifields.py

```diff
@@ -248,71 +246,34 @@ def test_get_k8s_pod_yaml(self, mock_pod_mutation_hook, redact):
             task = BashOperator(task_id="test", bash_command="echo hi")
 
         ti = TI(task=task, execution_date=EXECUTION_DATE)
-        rtif = RTIF(ti=ti)
 
-        # Test that pod_mutation_hook is called
-        mock_pod_mutation_hook.assert_called_once_with(mock.ANY)
+        render_k8s_pod_yaml = mock.patch.object(
+            ti, 'render_k8s_pod_yaml', return_value={"I'm a": "pod"}
+        ).start()
+
+        rtif = RTIF(ti=ti)
 
         assert ti.dag_id == rtif.dag_id
         assert ti.task_id == rtif.task_id
         assert ti.execution_date == rtif.execution_date
 
-        expected_pod_yaml = {
-            'metadata': {
-                'annotations': {
-                    'dag_id': 'test_get_k8s_pod_yaml',
-                    'execution_date': '2019-01-01T00:00:00+00:00',
-                    'task_id': 'test',
-                    'try_number': '1',
-                },
-                'labels': {
-                    'airflow-worker': 'worker-config',
-                    'airflow_version': version,
-                    'dag_id': 'test_get_k8s_pod_yaml',
-                    'execution_date': '2019-01-01T00_00_00_plus_00_00',
-                    'kubernetes_executor': 'True',
-                    'task_id': 'test',
-                    'try_number': '1',
-                },
-                'name': mock.ANY,
-                'namespace': 'default',
-            },
-            'spec': {
-                'containers': [
-                    {
-                        'args': [
-                            'airflow',
-                            'tasks',
-                            'run',
-                            'test_get_k8s_pod_yaml',
-                            'test',
-                            '2019-01-01T00:00:00+00:00',
-                        ],
-                        'image': ':',
-                        'name': 'base',
-                        'env': [{'name': 'AIRFLOW_IS_K8S_EXECUTOR_POD', 'value': 'True'}],
-                    }
-                ]
-            },
-        }
```

Comment:
This is gone,  as this is generated by code in TaskInstance so doesn't belong in a test in RTIF.